src/store.ts

```diff
@@ -19,6 +19,7 @@ interface NamedStoreOptions extends BaseStoreOptions {
 }
 
 export type StoreOptions = DeployStoreOptions | NamedStoreOptions
+type Key<T> = T extends string ? (T extends '' | `/${string}` ? never : T) : never
```

Comment:
if we want to go the route of stronger types, I'd love some input on naming conventions - this is more or less a POC/idea PR for consideration